packages/flutter/lib/src/widgets/overscroll_indicator.dart

```diff
@@ -198,7 +198,7 @@ class _GlowingOverscrollIndicatorState extends State<GlowingOverscrollIndicator>
         }
       }
     } else if (notification is ScrollEndNotification || notification is ScrollUpdateNotification) {
-      if (notification.dragDetails != null) { // ignore: undefined_getter
+      if ((notification as dynamic).dragDetails != null) {
```

Comment:
Please add "as dynamic" to the list of things that increase our technical debt:
https://github.com/flutter/flutter/blob/master/dev/devicelab/bin/tasks/technical_debt__cost.dart

Reply:
Done. Same cost as ignore.